### PR TITLE
Replace all remaining lang to language

### PR DESCRIPTION
**Pull request by ymo-sci**
BREAKING:
TenantData lang -> language
UserAccountData lang -> language
KnowledgeBaseEntry lang -> language

old getters/setters are deprecated but still available

KnowledgeBaseEntrys: execute `eod synchronize-knowledgebase` on console

For SQL Systems use e.g. this component-legacy.conf
```
mixing {
    legacy {
        SQLTenant {
            rename {
                tenantData_language: tenantData_lang
            }
        }
        SQLUserAccount {
            rename {
                userAccountData_language: userAccountData_lang
            }
        }
    }
}
```

For Mongo Systems use this renaming:
```
db.mongoUserAccount.updateMany( {}, { $rename: { "userAccountData_lang": "userAccountData_language" } } )
db.mongoTenant.updateMany( {}, { $rename: { "tenantData_lang": "tenantData_language" } } )
```

### CODE REVIEW

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/java/sirius/biz/tenants/TenantData.java [hunk 1]
```diff
@@ -558,8 +558,8 @@ public JournalData getJournal() {
         return journal;
     }
 
-    public LookupValue getLang() {
-        return lang;
+    public LookupValue getLanguage() {
```
Comment (sabieber) [inline src/main/java/sirius/biz/tenants/TenantData.java:561]: No deprecated variant?

**Review by sabieber — APPROVED** [review]

**Review by idlira — APPROVED** [review]

File: src/main/java/sirius/biz/tenants/TenantUserManager.java [hunk 2]
```diff
@@ -684,7 +686,7 @@ protected boolean checkApiToken(LoginData loginData, String givenApiToken) {
         }
 
         long currentTimestampInDays = TimeUnit.MILLISECONDS.toDays(System.currentTimeMillis());
-        // Timestamps of tomorrow and yesterday should be valid too, to be more gracefull with nightly scripts utilizing
+        // Timestamps of tomorrow and yesterday should be valid too, to be more gracefully with nightly scripts utilizing
```
Comment (jakobvogel) [inline src/main/java/sirius/biz/tenants/TenantUserManager.java:689]: Not sure, but I believe it should be _graceful_, as it is probably no adverb here. It does not describe "the art of being", but rather "the result of being".

**Review by jakobvogel — APPROVED** [review]

File: src/main/java/sirius/biz/tenants/TenantUserManager.java (hunk 2 above)
Comment (fhaScireum) [reply src/main/java/sirius/biz/tenants/TenantUserManager.java:689]: _graceful_ sounds better to me, too.

**Review by fhaScireum — APPROVED** [review]

**Review by cschierle — APPROVED** [review]

File: src/main/java/sirius/biz/tenants/TenantUserManager.java (hunk 2 above)
Comment (ymo-sci) [inline src/main/java/sirius/biz/tenants/TenantUserManager.java:689]:
```suggestion
        // Timestamps of tomorrow and yesterday should be valid too, to be more graceful with nightly scripts utilizing
```

**Review by andyHa — APPROVED** [review]

**Pull request merged**